.github/scripts/tp_hooks.py

```
@@ -0,0 +1,743 @@
+                print(f"  [{ticket_key}] SHADOW cost: {cost_summary}")
+            return
+
+        dest = Path(f"enhancements/{ep_slug}/TestPlan.md")
+        if not dest.parent.exists():
+            print(f"  [{ticket_key}] EP directory not found: {dest.parent}")
+            return
+
+        import shutil
+        shutil.copy2(output, dest)
+
+        subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)